### ISSUE TITLE
If I have multiple buckets, do I have to build multiple tusd services？

### ISSUE THREAD

**Issue by lshz0088** [tus/tusd#491]
**Question**
If I have multiple buckets, do I have to build multiple tusd services？

**Setup details**
Can different 'base path' be used to correspond to different 'buckets'`?

**Comment by Acconut** [comment]
> Can different 'base path' be used to correspond to different 'buckets'`?

No, that's not possible. Every tusd process can only upload to one bucket and this bucket has to be specific in advance using the -base-path option. I are able to work around this by uploading into one bucket and then copy the objects into different buckets once the upload is complete.

**Comment by vopolonc** [comment]
Is there Any update on it? @Acconut
I just trying to understand how to implement support of multiple regions, for example, separate buckets for EU and OAE

**Comment by Acconut** [comment]
Support for customizing buckets with the pre-create hook will be available once we finish https://github.com/tus/tusd/pull/1167.

**Comment by vansejer-ctrl** [comment]
Are there any advancements in this feature?

**Comment by Acconut** [comment]
@vansejer-ctrl Not yet unfortunately. I hope to resume work on https://github.com/tus/tusd/pull/1167 soon.
